doc/source/faq.rst

```diff
@@ -58,22 +40,257 @@ For example, to set the limit to 4 GB of memory::
+
+User for QAN monitoring
+-----------------------
+
+To add a local QAN instance,
+a command similar to the following is recommended:
+
+.. prompt:: bash
+
+   sudo pmm-admin add queries --user root --password root --create-user
+
+The superuser credentials are required only to set up the ``pmm-queries`` user
+with necessary privileges for collecting data.
+If you want to create this user yourself, the following privileges are required::
+
+ GRANT SELECT, PROCESS, SUPER ON *.* TO 'pmm-queries'@' localhost' IDENTIFIED BY 'pass' WITH MAX_USER_CONNECTIONS 5;
+ GRANT SELECT, UPDATE, DELETE, DROP ON performance_schema.* TO 'pmm-queries'@' localhost';
+
+If the ``pmm-queries`` user already exists,
+simply pass its credentials when you add the instance:
+
+.. prompt:: bash
+
+   sudo pmm-admin add queries --user pmm-queries --password pass
+
+For more information, run ``sudo pmm-admin add queries --help``.
+
+User for MySQL metrics monitoring
+---------------------------------
+
+To add a local MySQL metrics instance,
+a command similar to the following is recommended:
+
+.. prompt:: bash
+
+   sudo pmm-admin add mysql --user root --password root --create-user
+
+The superuser credential are required only to set up the ``pmm-mysql`` user
+with necessary privileges for collecting data.
+If you want to create this user yourself, the following privileges are required::
+
+ GRANT PROCESS, REPLICATION CLIENT ON *.* TO 'pmm-mysql'@'localhost' IDENTIFIED BY 'pass' WITH MAX_USER_CONNECTIONS 5;
+ GRANT SELECT ON performance_schema.* TO 'pmm-mysql'@'localhost';
+
+If the ``pmm-mysql`` user already exists,
+simply padd its credential when you add the instance:
+
+.. prompt:: bash
+
+   sudo pmm-admin add mysql --user pmm-mysql --password pass
+
+For more information, run ``sudo pmm-admin add mysql --help``.
+
+Can I monitor multiple MySQL instances?
+=======================================
+
+Yes, you can add multiple QAN and MySQL metrics monitoring instances
+on a single *PMM Client* (that is, ``queries`` and ``mysql`` types).
+In this case,
+you will need to provide a distinct port and socket for each instance
+using the ``--port`` and ``--socket`` variables,
+and specify a unique name for each instance
+(by default, it uses the host name).
+
+For example, if you are adding QAN and MySQL metrics monitoring instances
+for two local MySQL servers,
+the commands could look similar to the following:
+
+.. code-block:: bash
+
+   $ sudo pmm-admin add queries --user root --password root --create-user --port 3001 --socket /tmp/mysql-01.sock instance-01
+   $ sudo pmm-admin add queries --user root --password root --create-user --port 3002 --socket /tmp/mysql-02.sock instance-02
+   $ sudo pmm-admin add mysql --user root --password root --create-user --port 3001 --socket /tmp/mysql-01.sock instance-01
+   $ sudo pmm-admin add mysql --user root --password root --create-user --port 3002 --socket /tmp/mysql-02.sock instance-02
```

Comment:
`Flags --socket and --host are mutually exclusive.`
`Flags --socket and --port are mutually exclusive.`

So pmm-admin will not allow using both --port and --socket. Let's remove one of these.